app/server/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "lint": "eslint . --cache",
     "prebuild": "yarn && rm -rf build",
-    "build": "babel-node ../node_modules/.bin/webpack",
+    "build": "babel-node --presets es2015 $(npm bin)/webpack",
```

Comment:
`es2015` is a relatively large preset. The only reason we need it here for the `babel-node` runtime is to transpile the `imports`. Therefore, can we use the [ES2015 `import` plugin](https://babeljs.io/docs/plugins/transform-es2015-modules-commonjs/) instead here?